README.md

```diff
@@ -10,14 +10,19 @@ Hubot script for writing birthday messages to users. It uses [Tenor](https://ten
-* Reminds users of the upcoming birthdays one day and a few days in advance. It's possible to specify how long before the event occurs the reminder should be triggered.
-* Provides fault tolerance:
-  * if Tenor is for some reason unavailable right now, the script will try to request it a number of times with delays;
-  * if all the requests failed, users will receive their birthday messages anyway.
-* Creates private channels in advance and invites all the users to the channels except the birthday boys/girls for the purpose of discussing the presents for them. Expired channels are automatically removed.
+The functionality of the script is divided into two parts: handling birthdays and work anniversaries.
+- Birthdays
+  * Asking new employees to specify their date birth and reminding them about it if they ignore this request for some reason.
+  * Fetching a random GIF from [Tenor](https://tenor.com) before writing a birthday message to a birthday boy/girl. The `TENOR_SEARCH_TERM` environment variable stores a comma separated list of the tags which will be used when fetching GIFs. By default, the value of the variable is '[darthvaderbirthday](https://tenor.com/search/darthvaderbirthday),[futuramabirthday](https://tenor.com/search/futuramabirthday),[gameofthronesbirthday](https://tenor.com/search/gameofthronesbirthday),[harrypotterbirthday](https://tenor.com/search/harrypotterbirthday),[kingofthehillbirthday](https://tenor.com/search/kingofthehillbirthday),[lanadelreybirthday](https://tenor.com/search/lanadelreybirthday),[madhatterbirthday](https://tenor.com/search/madhatterbirthday),[pulpfictionbirthday](https://tenor.com/search/pulpfictionbirthday),[rickandmortybirthday](https://tenor.com/search/rickandmortybirthday),[rocketbirthday](https://tenor.com/search/rocketbirthday),[sheldonbirthday](https://tenor.com/search/sheldonbirthday),[simpsonbirthday](https://tenor.com/search/simpsonbirthday),[thesimpsonsbirthday](https://tenor.com/search/thesimpsonsbirthday),[tmntbirthday](https://tenor.com/search/tmntbirthday)'.
+  * Writing birthday messages to `#general`.
+  * Reminding users of the upcoming birthdays one day and a few days in advance. It's possible to specify how long before the event occurs the reminder should be triggered.
+  * Providing fault tolerance:
+    + if Tenor is for some reason unavailable right now, the script will try to request it a number of times with delays;
+    + if all the requests failed, users will receive their birthday messages anyway.
+  * Creating private channels in advance and inviting all the users to the channels except the birthday boys/girls for the purpose of discussing the presents for them. Expired channels are automatically removed.
+- Work anniversaries
+  * Memorizing the date when a new employee logs in to Rocket.Chat for the first time (the date is equal to the first working day in the company).
+  * Congratulating employees on anniversary of working in the company.
```

Comment:
For birthday we specify the channel for sending congratulations. probably we should do it here too.